docs/source/configuration/external.mdx

```diff
@@ -314,7 +314,7 @@ Control field changes are ignored with the exception of the control field. The c
 
 If the value of the "control" field is "Continue", then processing will continue as normal through the router. If the value is a map with a key of "Break" and a numeric status code, then the router will stop processing and return a response to the client. The data returned to the client is the "body" of the co-processor response to the router.
 
-If the Break code is a "success" code (2xx), then the body of the co-processor response should be valid data that the client is expecting. If the Break code is a "failure" code, then the body of the co-processor response will be formatted into an error response for the client.
+The body of the co-processor response should be valid graphql data that the client is expecting. If the Break code is a "failure" code, then the body of the co-processor response should contain a GraphQL errors array as shown in the following code snippet.
```

Comment:
```suggestion
The body of the co-processor response should be valid GraphQL data that the client is expecting. If the Break code is a "failure" code, then the body of the co-processor response should contain a GraphQL errors array as shown in the following code snippet.
```

Reply:
actually i will commit theese since i have edited the file in this pr already